.circleci/config.yml

```diff
@@ -230,6 +230,17 @@ jobs:
     environment:
       BROWSER: << parameters.browser >>
       BVER: << parameters.bver >>
+    trigger-qe-tests:
+      docker:
```

Comment:
NIT: you just want to execute the curl command right? see if you can use 
`executor: generic-executor`

in this docker file I have tried to limit the number of executors by declaring it upfront at top.